src/client/stylesheets/style.scss

```diff
@@ -282,15 +280,30 @@ a.contact-text:visited {
 
 
 /* react-select styling */
-.Select-control {
+.react-select__option {
+	color: #46433a !important;
+	background-color: white !important;
+	&:hover{
+		background-color: #deebff !important;
+		color: #333;
+	}
+}
+.react-select__menu {
+	z-index: 100 !important;
+}
+.react-select__control, .Select-control{
```

Comment:
It would be interesting to see why the `!important` is needed here.
What are they being overwritten by? Can we supersede them simply by having more selectors instead?
Usually it's not a good sign if we have to use !important and its use should be limited as much as possible.

Reply:
Since react-select use css in js to style its component, i don't think we can get rid of !important flag.